test/test_intensify/test_successive_halving.py

```diff
@@ -228,58 +306,26 @@ def target(x):
                         status=StatusType.SUCCESS, instance_id=i+1, seed=0,
                         additional_info=None)
 
-        # config2 & config3 should be capped and config1 should still be the incumbent
-        inc, _ = intensifier.intensify(challengers=[self.config2, self.config3],
-                                       incumbent=self.config1,
-                                       run_history=self.rh,
-                                       aggregate_func=average_cost)
+        intensifier._update_stage()
+
+        # config2 should be capped and config1 should still be the incumbent
+        inc, _ = intensifier.eval_challenger(challenger=self.config2,
+                                             incumbent=self.config1,
+                                             run_history=self.rh,
+                                             aggregate_func=average_cost)
 
         self.assertEqual(inc, self.config1)
-        self.assertEqual(self.stats.ta_runs, 2)
+        self.assertEqual(self.stats.ta_runs, 1)
         self.assertEqual(self.stats.inc_changed, 0)
         self.assertEqual(list(self.rh.data.values())[2][2], StatusType.CAPPED)
-        self.assertEqual(list(self.rh.data.values())[3][2], StatusType.CAPPED)
-
-    @attr('slow')
```

Comment:
Hm, what happened to this test?

Reply:
This functionality being tested in this is already being tested in the other test cases, so this was redundant.